init/kibana.conf

```diff
@@ -16,7 +16,6 @@ end script
 
 post-start script
   KB_HOME=/usr/local/kibana-4.1.4-linux-x64/
-  sh ${KB_HOME}/setDefaultIndex.sh
-  sh ${KB_HOME}/loadAssets.sh
-  sh ${KB_HOME}/refreshKibanaIndex.sh
+  /usr/bin/python ${KB_HOME}/setDefaultIndex.py
```

Comment:
We could. I was thinking about this in the spec file, but I couldn't do a ```cp -r ``` on the scripts directory since we have the buildRpm in there, so I just copied what I needed straight into the dir.